ethereum/contracts/relayer/coreRelayer/CoreRelayerDelivery.sol

```diff
@@ -223,18 +223,26 @@ abstract contract CoreRelayerDelivery is CoreRelayerGovernance {
         stack.transactionFeeRefundAmountPostForward = stack.transactionFeeRefundAmount;
         if (forwardInstructions.length > 0) {
             // If the user made a forward/multichainForward request, then try to execute it
-            stack.transactionFeeRefundAmountPostForward = emitForward(stack.transactionFeeRefundAmount, forwardInstructions);
+            stack.transactionFeeRefundAmountPostForward =
+                emitForward(stack.transactionFeeRefundAmount, forwardInstructions);
             status = DeliveryStatus.FORWARD_REQUEST_SUCCESS;
         } else {
             status = stack.callToTargetContractSucceeded
-                ? (stack.callToInstructionExecutorSucceeded ? DeliveryStatus.SUCCESS : DeliveryStatus.FORWARD_REQUEST_FAILURE)
+                ? (
```

Comment:
nit: I know this wasn't added in this pr, but we should not use nested ternary expressions. If more than one nesting is required, prefer an if/else chain